sbt-conductr/src/main/resources/LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Why is this license required here i.e. as part of resources? We have a LICENSE file which is all that is required.